### ISSUE TITLE
How to incorporate new build files?

### ISSUE THREAD

**Issue by BastianZim** [conda-forge/staged-recipes#13931]
Hi,

I was wondering what the suggested approach is to using new build files from conda-smithy for staged-recipes.

Specifically, I would like to test the changes incorporated in https://github.com/conda-forge/conda-smithy/pull/1441 in my local fork of `staged-recipes` using `./.scripts/run_docker_build.sh` but those changes are only available once a recipe is successfully converted into a feedstock, where they can be invoked with `build_locally.py`.
Is there a way to rerender `staged-recipes` to incorporate these changes, or do I have to manually copy those into my fork and then remove the changes again before submitting, once I have successfully tested the package?

Thanks!

**Comment by BastianZim** [comment]
Pinging @conda-forge/staged-recipes

**Comment by jakirkham** [comment]
Yeah staged-recipes doesn't re-render things as part of the build process. That would likely be pretty slow (takes ~2mins)

There might be other ways to add that functionality to staged-recipes. Is this something you are willing to take a look at?

cc @wolfv

**Comment by wolfv** [comment]
I have not yet had the time to look deeply at the CI scripts in staged recipes, but theoretically it should be enough to add `conda install ... boa` and run `conda mambabuild recipe` instead of `conda build recipe`.

I think it would be OK to do that inside the PR, but one would need to remove the changes before the PR is merged.

**Comment by BastianZim** [comment]
I went over it a bit more and would agree that just adding this manually and removing it later works. It's a bit of a hack since there are some parts that need to be fixed but it's possible. 
Regarding adding this to staged-recipes; I think that's a bit more difficult since there are some jinja templates used and the build process for staged recipes is in a python script whereas later it's in a shell script (If I'm not mistaken, I just had a cursory look at best 😄). 

> There might be other ways to add that functionality to staged-recipes. Is this something you are willing to take a look at?

I can definitely try but this would be a bit above my abilities I must admit...
